app/components/spotlight/analytics/dashboard_component.rb

```diff
@@ -9,7 +9,63 @@ class DashboardComponent < ViewComponent::Base
       def initialize(current_exhibit:)
         super
         @current_exhibit = current_exhibit
-        @dates = { 'start_date' => '365daysAgo', 'end_date' => 'today' }
+        @default_start_date = 1.year.ago.to_date.to_s
+        @default_end_date = Time.zone.today.to_date.to_s
```

Comment:
Things seem to get a little wonky if the configured max/min dates fall outside this range. The UI displays the range set here instead of the configured min/max which is probably unexpected.

Reply:
The range set here is a year. The max/min are there to make sure people can't set a date using the drop downs that the admin doesn't want them to set. It doesn't set the default range.

Reply:
But if I set my app's config to a start_date that's less than a year ago when I first load the analytics page it's going to try to show the last year even though that's outside the restricted range I can select within the app. That seems odd.